crates/router/src/types/domain/event.rs

```diff
@@ -19,24 +19,58 @@ use crate::{
 
 #[derive(Clone, Debug, router_derive::ToEncryption)]
 pub struct Event {
+    /// A string that uniquely identifies the event.
     pub event_id: String,
+
+    /// Represents the type of event for the webhook.
     pub event_type: EventType,
+
+    /// Represents the class of event for the webhook.
     pub event_class: EventClass,
+
+    /// Indicates whether the current webhook delivery was successful.
     pub is_webhook_notified: bool,
+
+    /// Reference to the object for which the webhook was created.
     pub primary_object_id: String,
+
+    /// Reference to the object type for which the webhook was created.
     pub primary_object_type: EventObjectType,
+
+    /// The timestamp when the webhook was created.
     pub created_at: time::PrimitiveDateTime,
+
+    /// Merchant identifier to which the webhook was sent.
     pub merchant_id: Option<common_utils::id_type::MerchantId>,
+
+    /// Business Profile identifier to which the webhook was sent.
     pub business_profile_id: Option<common_utils::id_type::ProfileId>,
```

Comment:
These actually are the identifiers of the merchant account and business profile the object is associated with.